cmd/make-bm-worker/main.go

```diff
@@ -26,6 +26,7 @@ metadata:
   name: {{ .Name }}
 spec:
   online: true
+  hardwareProfile: {{ .HardwareProfile }}
```

Comment:
You can make this conditional where hardwareProfile doesn’t show up at all if it’s empty. See some examples in the dev-scripts version